python-avd/pyavd/_eos_designs/schema/schema_fragments/network_ports.schema.yml

```diff
@@ -20,6 +20,18 @@ keys:
             The regular expression must match the full hostname.
           items:
             type: str
+        platforms:
```

Comment:
let's add a description on the `network_ports` stating that when `switches` and `platforms` are both set, then only devices matching both criteria will be configured.